desktop/windows/release-notes/index.md

```diff
@@ -32,6 +32,11 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
 > .button .primary-btn }
 
+### Security
+
+- Fixed [CVE-2022-23774](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23774) where Docker Desktop allows attackers to move arbitrary files.
+- Fixed [CVE-2021-44719](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44719) where Docker Desktop using the Hyper-V backend could be used to access any user file on the host from a container, bypassing the allowed list of shared folders.
```

Comment:
Are we sure this issue occurred on Windows too? I know the grpcfuse change also applies to Hyper-V, but I thought that the security issue was only on Mac. I'm not 100% sure though.
(Ref: https://docs.google.com/document/d/1ZqXppY5pIdms21ird3O7VvqHrqY6efIFiPm6SuMfOYg/edit?usp=sharing)

Reply:
I wondered as well. The docs says Mac Intel only, so maybe let’s stick with that.
@djs55 would know 😁